packages/next/taskfile.js

```diff
@@ -23,20 +23,25 @@ const externals = {
   chalk: 'chalk',
   'node-fetch': 'node-fetch',
 
+  // css-loader
+  postcss: 'postcss',
+
+  // sass-loader
+  // (also responsible for these dependencies in package.json)
+  'node-sass': 'node-sass',
+  sass: 'sass',
+  fibers: 'fibers',
+  klona: 'klona',
```

Comment:
I don't understand why this is necessary if `sass-loader` isn't compiled. Can you elaborate please?

Reply:
Yes these are placeholders for the loader inlining, I'm first working on getting the tests to pass without the loader inlining.